metrics/web/lib/dashboard/presentation/models/dashboard_page_parameters_model.dart

```diff
@@ -0,0 +1,53 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:metrics/common/presentation/navigation/models/page_parameters_model.dart';
+import 'package:metrics/dashboard/presentation/pages/dashboard_page.dart';
+
+/// A class, that represents [DashboardPage] parameters parsed from
+/// query parameters.
+class DashboardPageParametersModel implements PageParametersModel {
+  /// A [String], that represents a name of the project.
+  final String projectName;
```

Comment:
🔖
```suggestion
  final String projectFilter;
```
Would this name describe this parameter a bit better? What do you think?